No rule to make target 'functional/block_cloning/block_cloning_copyfilerange_cross_dataset.ksh', needed by 'all-am'.  Stop.

### System information
Distribution Name	| Devuan
Distribution Version	| 5 Daedalus (Debian 12 fork)
Kernel Version	| 6.1.0-33-amd64
Architecture	| x64
OpenZFS Version	| 2.2.7

### Describe the problem you're observing
error while building 2.2.7 zfs 

### Include any warning/errors/backtraces from the system logs
```
freeartist-devuan@devuan:~/Downloads/zfs/zfs-2.2.7$ make -j4
  GEN      gitrev
make  all-recursive
make[1]: Entering directory '/home/freeartist-devuan/Downloads/zfs/zfs-2.2.7'
Making all in include
make[2]: Entering directory '/home/freeartist-devuan/Downloads/zfs/zfs-2.2.7/include'
make[2]: Nothing to be done for 'all'.
make[2]: Leaving directory '/home/freeartist-devuan/Downloads/zfs/zfs-2.2.7/include'
Making all in tests/zfs-tests/tests
make[2]: Entering directory '/home/freeartist-devuan/Downloads/zfs/zfs-2.2.7/tests/zfs-tests/tests'
make[2]: *** No rule to make target 'functional/block_cloning/block_cloning_copyfilerange_cross_dataset.ksh', needed by 'all-am'.  Stop.
make[2]: Leaving directory '/home/freeartist-devuan/Downloads/zfs/zfs-2.2.7/tests/zfs-tests/tests'
make[1]: *** [Makefile:12343: all-recursive] Error 1
make[1]: Leaving directory '/home/freeartist-devuan/Downloads/zfs/zfs-2.2.7'
```

--- a/configure.ac
+++ b/configure.ac
@@ -41,7 +41,7 @@ AC_CONFIG_MACRO_DIR([config])
 AC_CANONICAL_TARGET
 AM_MAINTAINER_MODE
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
-AM_INIT_AUTOMAKE([subdir-objects foreign])
+AM_INIT_AUTOMAKE([subdir-objects foreign tar-pax])
 # Remove default macros from config.h:
 #   PACKAGE, PACKAGE_{BUGREPORT,NAME,STRING,TARNAME,VERSION}, STDC_HEADERS, VERSION
 AC_CONFIG_HEADERS([zfs_config.h], [

